static/js/components/dashboard/FinancialAidCard_test.js

```diff
@@ -122,7 +122,7 @@ describe("FinancialAidCard", () => {
       program.financial_aid_user_info.has_user_applied = false;
       let wrapper = renderCard({ program });
       wrapper.find('.full-price').simulate('click');
-      assert.ok(setSkipDialogStub.calledWith(true), 'Dialog should get opened');
+      sinon.assert.calledWith(setSkipDialogStub, true);
```

Comment:
I think it's nice to have the `assert.ok` message - if you break a test it gives you quick feedback about what didn't happen.